apps/src/netsim/netsim.js

```diff
@@ -504,7 +502,8 @@ NetSim.prototype.synchronousDisconnectFromShard_ = function () {
   this.myNode.stopSimulation();
   this.myNode.synchronousDestroy();
   this.myNode = null;
-  this.shardChange.notifyObservers(null, null);
+  // Don't notify observers, this should only be used when navigating away
+  // from the page.
```

Comment:
Removed because doing this notification during synchronous disconnect causes the client to try and recover/reconnect while you're navigating away from the page!  Which is no good, and might have been the cause of a bunch of the extra "[New Node]" instances we were seeing.

Downside of removing this line is that you don't _see_ the simulator back out to the lobby as you're navigating away from the page, but I'm not sure there was a real benefit to that anyway.